src/runtime_src/core/common/api/xrt_kernel.cpp

```diff
@@ -2593,6 +2593,24 @@ class run_impl
   {
     return cmd->get_ert_packet();
   }
+
+  [[nodiscard]] std::vector<char>
```

Comment:
There is no harm in discarding this return value.   I suggest you remove [[nodiscard]].